packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -967,7 +967,6 @@ async function processRequest(params) {
       log.error(
         'Internal Reconciliation Reports are no longer valid, as Cumulus is no longer utilizing Elasticsearch'
       );
-      //TODO remove internal rec report code
```

Comment:
was the actual internal rec report code removed in a previous PR and this TODO is vestigial? I checked that it is actually not referenced in the code but I just want to double check what happened here

Reply:
This note is a good portion of what this PR is doing - we removed the call *to* the rec report code in PR 1, but some of the methods referenced were still used until PR2.      This PR is to clean up/remove the unused internal rec report and associated lib methods.